openapi.yaml

```diff
@@ -599,6 +599,273 @@ paths:
+            view 123E4567-E89B-12D3-A456-426614174000
+    delete:
+      x-linode-grant: unrestricted only
+      tags:
+      - account
+      summary: Entity Transfer Cancel
+      description: >
+        Cancels the Entity Transfer for the provided token. An email notification is sent to the account that created
+        this transfer. Transfers can not be canceled if they are expired or have been accepted.
+
+
+        This command can only be accessed by the unrestricted users of the accounts that created this transfer.
+      operationId: deleteEntityTransfer
+      x-linode-cli-action: delete
+      security:
+      - personalAccessToken: []
+      - oauth:
+        - account:read_write
+      responses:
+        '200':
+          description: >
+            Entity Transfer canceled.
+          content:
+            application/json:
+              schema:
+                type: object
+        default:
+          $ref: '#/components/responses/ErrorResponse'
+      x-code-samples:
+      - lang: Shell
+        source: >
+          curl -H "Authorization: Bearer $TOKEN" \
+            -X DELETE \
+            https://api.linode.com/v4/account/entity-transfers/123E4567-E89B-12D3-A456-426614174000
+      - lang: CLI
+        source: >
+          linode-cli account entity-transfers \
+            delete 123E4567-E89B-12D3-A456-426614174000
+  /account/entity-transfers/{token}/accept:
+    x-linode-cli-command: entity-transfers
+    parameters:
+      - name: token
+        in: path
+        description: The UUID of the Entity Transfer.
+        required: true
+        schema:
+          type: string
+          format: uuid
+    post:
+      x-linode-grant: unrestricted only
+      tags:
+      - Account
+      summary: Entity Transfer Accept
+      description: >
+        Accept an Entity Transfer for the provided token to receive the resources included in the transfer to your
+        account. A transfer must have a `pending` status in order to be accepted. Transfers can take several hours to
+        complete. Email notifications are sent to the accounts that created and accepted this transfer.
+
+
+        This command can only be accessed by the unrestricted users of the account that receives the transfer. Users
+        of the same account that created a transfer cannot accept the transfer.
+
+
+        There are several conditions that must be met in order to successfully create a transfer request:
+          1. The account accepting the transfer must have a registered payment method and must not have a past due
+          balance or other account limitations for the entities to be transferred.
+
+          1. The entity must be owned by the account that created the transfer and must not be assigned to another
+          Entity Transfer or have any active Terms of Service violations.
+
+          1. Linodes must not be assigned to a NodeBalancer, Firewall, or Managed Service, have any attached Block
```

Comment:
I'll repeat my request for a bullet sub-list here as I did above as well.

Reply:
For a lot of the things in this bullet, the transfer creator had to have added those relationships between creating the transfer and the receiver attempting to accept it, so the creator can act to repair this block. Not sure if we want to mention that or what words would do it best.

Reply:
Added this language to that effect @alsaihn :

"Any and all of the above conditions must be cured and maintained by the relevant account prior to the transfer's expiration to allow the transfer to be accepted by the receiving account."